src/search/landmarks/landmark_status_manager.cc

```diff
@@ -51,27 +51,27 @@ static vector<pair<const LandmarkNode *, vector<const LandmarkNode *>>> get_reas
 }
 
 LandmarkStatusManager::LandmarkStatusManager(
-    LandmarkGraph &graph,
-    bool progress_goals,
-    bool progress_greedy_necessary_orderings,
-    bool progress_reasonable_orderings)
-    : lm_graph(graph),
-      goal_landmarks(progress_goals ? get_goal_landmarks(graph)
+    LandmarkGraph &landmark_graph,
+    const bool progress_goals,
+    const bool progress_greedy_necessary_orderings,
+    const bool progress_reasonable_orderings)
```

Comment:
I'm not sure those consts are needed here since it's anyways a copy and not a reference, but we will discuss the use of const anyways in the next Fast Downward meeting, so let's see afterwards.

Reply:
I put it there because my IDE recommended it, but to be honest I don't know what difference it makes. (The IDE didn't want to tell me.) I'm fine with removing `const` if we end up deciding that we don't want to overload the code with `const` qualifiers. There's plenty more places where this becomes relevant, though, so I will probably have to go through the entire landmark code again to check which `const`s we actually want.